Dockerfile

```diff
@@ -66,11 +66,11 @@ ARG CHARTS_REPO="https://rke2-charts.rancher.io"
 ARG CACHEBUST="cachebust"
 COPY charts/ /charts/
 RUN echo ${CACHEBUST}>/dev/null
-RUN CHART_VERSION="v3.13.3"     CHART_FILE=/charts/rke2-canal-chart.yml             CHART_BOOTSTRAP=true    /charts/build-chart.sh
-RUN CHART_VERSION="1.10.101"    CHART_FILE=/charts/rke2-coredns-chart.yml           CHART_BOOTSTRAP=true    /charts/build-chart.sh
-RUN CHART_VERSION="1.36.300"    CHART_FILE=/charts/rke2-ingress-nginx-chart.yml     CHART_BOOTSTRAP=false   /charts/build-chart.sh
-RUN CHART_VERSION="v1.18.8"     CHART_FILE=/charts/rke2-kube-proxy-chart.yml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
-RUN CHART_VERSION="2.11.100"    CHART_FILE=/charts/rke2-metrics-server-chart.yml    CHART_BOOTSTRAP=false   /charts/build-chart.sh
+RUN CHART_VERSION="v3.13.3"     CHART_FILE=/charts/rke2-canal.yaml             CHART_BOOTSTRAP=true    /charts/build-chart.sh
```

Comment:
We've lopped off the `-chart` infix to make the `--disable` logic work? There was a reason the infix made sense to me but I can't remember it now. Hopefully it isn't load-bearing.

Reply:
I couldn't find anything that depends on it. Finding the list of things that can be disabled is already difficult enough, given how we package and distribute the charts. I was hoping to just point users at the rke2-charts site, as described in the PR notes, but the chart names there don't match because we're appending -chart to the names here.

Reply:
Another option would be to template the names in at build time, but even that is a bit of a hack as we don't actually have a fixed list of items - it's just whatever is in the charts subdir of the runtime image.

Reply:
I think your first instinct for a "document fix" is probably good enough. If it turns out not to be we can generate a chart inventory at build-time similar to how we do for images.